ws-nextjs-app/pages/[service]/watch/[id]/LiveTvPageLayout.tsx

```diff
@@ -22,8 +22,60 @@ export default function LiveTvLayout({ pageData }: LiveTVPageProps) {
     pageData = liveTvFixture;
   }
 
+  const renderCuration = ({ curation }: { curation: Curation }) => {
+    const {
+      summaries,
+      curationId,
+      title: curationTitle,
+      link,
+      position,
+      ...curationProps
+    } = curation;
+    return (
+      <React.Fragment key={`${curationId}-${position}`}>
+        <LiveTVCuration
+          summaries={summaries || []}
+          title={curationTitle}
+          position={position}
+          link={link}
+          // curationLength={curations.length}
+          renderVisuallyHiddenH2Title={position === 0}
+          curationId={curationId}
+          {...curationProps}
+        />
+      </React.Fragment>
+    );
+  };
+
   const { curations, description, title } = pageData;
 
+  const mediaCollectionCuration = curations?.find(
+    curation => curation.mediaCollection,
+  );
+
+  const radioScheduleCuration = curations?.find(
+    curation => curation.radioSchedule,
+  );
+
+  const portraitVideoCuration = curations?.find(
+    curation => curation.portraitVideo,
+  );
```

Comment:
I don't think we should cherry pick these components from the data. 

If possible, can we remove the curation with the media collection from the list of curations, and then just map over the rest of the curations to render it? This means the page is more flexible for the future & can just render whatever data we get back from the BFF